common/workunit/workunit.ipp

```diff
@@ -371,7 +371,7 @@ public:
     void setState(WUState state);
     void setStateEx(const char * text);
     void setAgentSession(__int64 sessionId);
-    void setSecurityToken(const char *value);
+    bool setDistributedAccessToken(const char * wuid, const char * user);
```

Comment:
I really wish this method could dissapear.  If not, whey can't the wuid and user come from the workunit?